### DISCUSSION TITLE
unable to find entities, while using cli (inside docker)

### DISCUSSION THREAD

**Discussion by ahhmarr** [mikro-orm/mikro-orm#5923] [Q&A]
I'm encountering the following issue while using CLI (migrate/seed) 

`MetadataError: Only abstract entities were discovered, maybe you forgot to use @Entity() decorator?`

Note: this seems to be only when I'm using CLI and not via the code 

repo to reproduce

https://github.com/ahhmarr/mikro-orm-enitities-issue

**Comment by B4nan** [comment]
This usually means the decorators are not executed properly. Also, you should always try the latest version.

What do you get from inside the container when you run `npx mikro-orm debug`?

**Reply by ahhmarr** [reply]
Same issue on latest version as well (updated the repo)

I see from debug it doesn't get any entities 
```
- dependencies:
   - mikro-orm 6.3.4
   - node 20.16.0
   - knex 3.1.0
   - pg 8.12.0
   - typescript 5.5.4
 - package.json found
 - ts-node enabled
 - searched config paths:
   - /app/src/database/mikro-orm.config.ts (found)
   - /app/dist/database/mikro-orm.config.js (found)
   - /app/src/mikro-orm.config.ts (not found)
   - /app/mikro-orm.config.ts (not found)
   - /app/dist/mikro-orm.config.js (not found)
   - /app/mikro-orm.config.js (not found)
 - configuration found
 - database connection successful
 - will use `entities` array (contains 0 references and 1 paths)
   - /app/dist/entity/*entity.js (found)
 - could use `entitiesTs` array (contains 0 references and 1 paths)
   - /app/src/entity/*entity.ts (found)
   ```

**Reply by B4nan** [reply]
Nope, that's not what this means, it says there. is one path and it was able to find some files matching the pattern, in both prod and dev/ts modes. The CLI should prefer the TS entities/configs, and it has TS support enabled. The logs all look good to me.

**Reply by ahhmarr** [reply]
aah i misread then 

But still running `npx mikro-orm seeder:run` gives the same error

**Comment by mikelgmh** [comment]
Same error here. Debugging mikro-orm shows that everything is fine.

```
- will use `entities` array (contains 0 references and 1 paths)
   - C:/Users/me/Documents/projects/test/**/*.entity.ts (found)
```

**Reply by B4nan** [reply]
You should have compiled/JS paths in `entities` option, while you have the source/TS paths there instead.
